tamara_gadieva/graph.hpp

```diff
@@ -67,18 +71,31 @@ class Graph {
 
   void update_vertex_depth(const VertexId& id, const int depth);
 
+  const std::vector<EdgeId>& get_colored_edges(const Edge::Color& color) const;
+
+  std::vector<VertexId> get_vertices_ids_in_depth(int depth);
+
   const std::vector<VertexId>& get_vertices_ids_in_depth(int depth) const;
 
   const std::vector<EdgeId>& get_edge_ids(const VertexId& id) const;
 
   const int get_depth() const { return vertices_ids_in_depth_.size(); }
 
+  const int get_vertices_in_depth_number(int depth) const {
+    return vertices_ids_in_depth_[depth].size();
+  }
```

Comment:
Обсуждали. что гибким интерфейсом будет - возвращать список граней
```suggestion
  const std::vector<EdgeId>& get_vertex_ids_in_depth(int depth) const {
    return vertices_ids_in_depth_[depth];
  }
```